src/08_comprehensions.py

```diff
@@ -12,13 +12,19 @@
 
 y = []
 
+for i in range(1, 6):
+    y.append(i)
+
 print (y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
 y = []
 
+for i in range(10):
```

Comment:
These work, of course, but the problem is list comprehensions and these are...not list comprehensions :)

They're looking for `y = [i * i * i for i in range(10)]`

Reply:
So the distinction is that a list comprehension always occurs within the list reference?

Reply:
As far as I can tell, you're right and it's syntactic sugar for list creation, no extra functionality, so I definitely won't make you rewrite them On the off chance you ever get asked about them, you'll know :)